public/assets/images/icons/icon-warning.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22px" height="22px" viewBox="0 0 22 22">
+<path fillrule="evenodd" d="M 11 22C 4.93 22-0 17.07-0 11-0 4.93 4.93 0 11 0 17.07 0 22 4.93 22 11 22 17.07 17.07 22 11 22ZM 11 1.76C 5.91 1.76 1.76 5.91 1.76 11 1.76 16.09 5.91 20.24 11 20.24 16.09 20.24 20.24 16.09 20.24 11 20.24 5.91 16.09 1.76 11 1.76ZM 11 18.04C 10.27 18.04 9.68 17.45 9.68 16.72 9.68 15.99 10.27 15.4 11 15.4 11.73 15.4 12.32 15.99 12.32 16.72 12.32 17.45 11.73 18.04 11 18.04ZM 11 13.64C 10.51 13.64 10.12 13.25 10.12 12.76 10.12 12.76 10.12 4.4 10.12 4.4 10.12 3.91 10.51 3.52 11 3.52 11.49 3.52 11.88 3.91 11.88 4.4 11.88 4.4 11.88 12.76 11.88 12.76 11.88 13.25 11.49 13.64 11 13.64Z" fill="#1d2022"></path></svg>
```

Comment:
new line, check your configurations, editors usually have an option to put an empty line at the end if it's missing